streams/writable-streams/aborting.js

```diff
@@ -204,34 +209,64 @@ promise_test(t => {
 }, 'Aborting a WritableStream puts it in an errored state, with a TypeError as the stored error');
 
 promise_test(t => {
-  const ws = new WritableStream();
+  let resolveWritePromise;
```

Comment:
Similar to the first test this appears to have been working due to async events not being processed so the stream isn't fully starting, or calling write in response to the writer's write being called. This ends up queuing a write (waiting for the start) and the test ends up testing what it was trying to. However it doesn't look like anything (other than implementation details) should actually prevent the stream from constructing, write from being called (and completing), then abort being called (but all the previous calls completed before abort happened).

I guess this could also be achieved by having start() return a promise rather than seed the test with an initial write, but it should be testing the same behavior just in a more deterministic manner and not relying on events not yet being processed by the browser.